src/store/hooks.ts

```diff
@@ -1,5 +1,5 @@
 import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux';
-import type { RootState, AppDispatch } from './store';
+import { AppDispatch, RootState } from 'store/store';
```

Comment:
store/store is not a good path for import, you can name store.ts file as index.ts and your import path will be only 'store'